File: chroma-league/src/test/java/net/booone/chromaleague/hud/animations/LoseAnimationTest.java
package net.booone.chromaleague.hud.animations;

import net.booone.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class LoseAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playLoseAnimation() {
        new AnimationTester().testAnimation(new LoseAnimation(), 100);
    }
}

File: test/hud/animations/LoseAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {LoseAnimation} from '../../../src/hud/animations/LoseAnimation.js';

/**
 * Integration test for defeat animation on actual hardware.
 */
describe('LoseAnimation Integration', () => {
    test('playLoseAnimation', async () => {
        await new AnimationTester().testAnimation(new LoseAnimation(), 100);
    }, 30000);
});
